src/python/views/demo_01_fancy/__init__.py

```diff
@@ -3,7 +3,8 @@
 from graphistry import PyGraphistry
 from css import all_css
 from time import sleep
-from util import getChild
+# import util
+from util.log import getChild
```

Comment:
demos should normally just be `import logging; logger = logging.getLogger(__name__)`